orchestrate/local/my_airflow/requirements.txt

```diff
@@ -32,7 +32,7 @@ apache-airflow-providers-sqlite==3.7.1
 apache-airflow-providers-ssh==3.10.1
 apache-airflow-providers-tableau==4.4.2
 packaging>=23.2
-python-jose==3.3.0
+python-jose>=3.4.0
```

Comment:
@BAntonellini can you check the Datacoves defaults for this, Github suggested we upgrade
cc/ @sconley-datacoves